crt_portal/cts_forms/templates/forms/complaint_view/show/index.html

```diff
@@ -20,10 +20,23 @@ <h2 class="margin-right-205 margin-top-0 margin-bottom-0 backend-blue">
             {{ data.status }}
           </span>
         </div>
-        <p>
-          <span class="details-date-label text-uppercase backend-blue">Received: </span>
-          <span class="backend-blue">{{ data.create_date|date:'g:i a F j, Y' }}</span>
-        </p>
+        <p class="display-flex">
+          <span>
+            <span class="details-date-label text-uppercase backend-blue">Received: </span>
+            <span class="backend-blue">{{ data.create_date|date:'g:i a F j, Y' }}</span>
+          </span>
+          {% if data.intake_format == 'web' %}
+            <img src="{% static "img/intake-icons/ic_web.svg" %}" class="icon margin-left-2" alt="">
+          {% elif data.intake_format == 'email' %}
+            <img src="{% static "img/intake-icons/ic_email.svg" %}" class="icon margin-left-2" alt="">
```

Comment:
adding the empty alt tags tells screen readers this is an icon they don't have to care about. We might want to consider adding an `sr-only` tag that reads out `Intake method: web` to screen readers?

Reply:
I think adding the description makes the most sense since there is no accompanying text. Great catch!